Components/Metrics/PolydataDummyPenalty/elxPolydataDummyPenalty.hxx

```diff
@@ -270,6 +275,10 @@ template <class TElastix>
 unsigned int
 PolydataDummyPenalty<TElastix>::ReadMesh(const std::string & meshFileName, typename FixedMeshType::Pointer & mesh)
 {
+#ifdef __wasm32__
+  log::error(std::ostringstream{} << "File IO not supported in WebAssembly builds.");
```

Comment:
Please pass the string directly to log::error, without `std::ostringstream{} <<`.  (ostringstream is only necessary when a string needs to be built by `<<` insertions). So please just:

    log::error("File IO not supported in WebAssembly builds.");

Reply:
BTW, maybe an `itkExceptionMacro("File IO not supported in WebAssembly builds.");` would be more appropriate, right?

Reply:
Yes, we could do both -- WebAssembly has limited C++ exception support at the moment, so it will help to also log an error message.

Reply:
Thanks, can you please explain what you mean by _limited C++ exception support_?

Reply:
The current Emscripten toolchain adds a JavaScript shim for C++ exceptions, and I added a shim for C++ exceptions in WASI that calls `abort();`. There is newer, experimental WASM-exceptions in some newer toolchains, but I have not enabled them by default.